sync/sync.go

```diff
@@ -154,6 +155,43 @@ func (s *String) SetString(val string) error {
 	return nil
 }
 
+// TimeDuration is Time.Duration type with concurrent access support.
+type TimeDuration struct {
+	rw    sync.RWMutex
+	value time.Duration
+}
+
+// Get returns the internal value.
+func (s *TimeDuration) Get() time.Duration {
```

Comment:
As codecov mentioned maybe we should write some unit tests to cover the methods for TimeDuration.